wicketd/src/bin/wicketd.rs

```diff
@@ -23,6 +23,9 @@ enum Args {
 
         #[clap(short, long, action)]
         address: SocketAddrV6,
+
+        #[clap(long, action)]
+        artifact_address: SocketAddrV6,
```

Comment:
Could we document the different between these two? It's obvious in the context of this PR, less obvious when read in isolation

Reply:
Done